tests/ed25519.rs

```diff
@@ -281,7 +285,7 @@ mod serialisation {
     #[test]
     fn serialize_signature_size() {
         let signature: Signature = Signature::from_bytes(&SIGNATURE_BYTES).unwrap();
-        assert_eq!(serialized_size(&signature) as usize, 72); // These sizes are specific to bincode==1.0.1
+        assert_eq!(serialized_size(&signature) as usize, 64); // These sizes are specific to bincode==1.0.1
```

Comment:
The `serde` serializers used by the `ed25519` crate are slightly different from the ones currently in `ed25519_dalek` .

Namely, they serialize a signature as a 64-element tuple, and in doing so avoid a length prefix when using formats like `bincode`, hence the signature size changed.